app/modules/audioplayer/collections/play_queue.collection.ts

```diff
@@ -25,17 +25,22 @@ export class PlayQueue extends BaseCollection<Model> {
           status: 'NULL',
           id: 176755192,
           stream_url: 'https://api.soundcloud.com/tracks/176755192/stream'
+        },
+        {
+          status: 'NULL',
+          id: 176724778271555192,
+          stream_url: 'https://api.soundcloud.com/tracks/247782715/stream'
         }
       ]);
       return PlayQueue.instance;
     }
   }
 
-  comparator(item: PlayQueueItem) {
+  static comparator(item: PlayQueueItem): Array<any> {
```

Comment:
Why did you set the method static? It is used by backbone to sort the items in the collection so it must not be static